source/watchdog.py

```diff
@@ -78,8 +78,20 @@ def alive():
 	windll.kernel32.ResetEvent(_cancelCallEvent)
 	# Set the timer so the watcher will take action in MIN_CORE_ALIVE_TIMEOUT
 	# if this function or asleep() isn't called.
-	windll.kernel32.SetWaitableTimer(_coreDeadTimer,
-		ctypes.byref(ctypes.wintypes.LARGE_INTEGER(-int(10000000 * MIN_CORE_ALIVE_TIMEOUT))),
+	SECOND_TO_100_NANOSECOND = 10000000  # nanosecond is 10^9, 10^7 is hundreds of nanoseconds

```

Comment:
I think this should be changed for readability

```suggestion
	SECOND_TO_100_NANOSECOND = 10**7  # nanosecond is 10^9, 10^7 is hundreds of nanoseconds
```

Or

```suggestion
	SECOND_TO_100_NANOSECOND = 10_000_000  # nanosecond is 10^9, 10^7 is hundreds of nanoseconds
```

Reply:
Yep, good point.